modules/RTC/RTC.js

```diff
@@ -278,6 +278,23 @@ export default class RTC extends Listenable {
         // Add Last N change listener.
         this.addListener(RTCEvents.LASTN_ENDPOINT_CHANGED,
             this._lastNChangeListener);
+
+        this.addListener(
+            RTCEvents.SELECTED_UPDATE,
+            isSelected => {
+                const jvbSession = this.conference.jvbJingleSession;
+
+                if (jvbSession) {
+                    jvbSession.peerconnection.setIsSelected(isSelected);
```

Comment:
I don't like how it uses private variables, but I guess you have figured out how to add the lest code.
Is it possible that SELECTED_UPDATE would arrive before jvbSession is ready and would we care (over WebSocket data channel for example ) ?

Reply:
well this isn't technically a private variable (or at least it's not indicated that it is, by being named with a leading `_`), but I did look into other ways (that would have been 'cleaner') but they would require a lot of work and justifying it for this (where this solution wasn't _that_ bad) didn't seem right.

Reply:
as far as getting the update before the session is ready, i don't think we care too much since this can be viewed as a best effort thing anyway.  since we default `isSelected` to true, i don't think this scenario would cause the endpoint to send low quality when we need high, if anything it would continue to send high when we only needed low which isn't so bad.

Reply:
Actually I gave this part second thought and why can't we move it to JitsiConference ? I don't like that now RTC module has knowledge about JitsiConference, JingleSessionPC and is in fact dealing with kind of internal parts of JitsiConference. It should not be a big change.

Reply:
In fact I wanted to get rid of this.conference field in RTC. It's used in two places both are kind of hacky.

Reply:
Or even JingleSessionPC (the non p2p one) could be listening to that even directly - it has the RTC module instance.

Reply:
> Or even JingleSessionPC (the non p2p one) could be listening to that even directly - it has the RTC module instance.

This would be nice...It didn't listen to any RTCEvents before so it didn't seem consistent to add that there, but I will try that out and update here if you think that'll be alright.